kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionImpl.scala

```diff
@@ -289,7 +289,7 @@ class KyuubiSessionImpl(
   @volatile private var engineLastAlive: Long = _
   val engineAliveTimeout = sessionConf.get(KyuubiConf.ENGINE_ALIVE_TIMEOUT)
   val aliveProbeEnabled = sessionConf.get(KyuubiConf.ENGINE_ALIVE_PROBE_ENABLED)
-  var engineAliveMaxFailCount = 3
+  var engineAliveMaxFailCount = sessionConf.get(KyuubiConf.ENGINE_ALIVE_MAX_FAIL_COUNT)
```

Comment:
You can fix these warning by the way

![image](https://github.com/apache/kyuubi/assets/3898450/3038a3dd-ed29-4ee9-a327-d851fac14844)